### PR TITLE
PLAT-111125: ImageItem resizing issues

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
Fixed ImageItem resizing issues. Added scaling to parent div in sampler and imported Image instead of ImageBase for the imageComponent prop, as Image component is wrapped in the ResponsiveImageDecorator which fixes the problem concerning the updating of the image to reflect the new sizing.

### Links
PLAT-111125

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: samples/sampler/stories/default/ImageItem.js [hunk 1]
```diff
@@ -23,22 +23,19 @@ export default {
...
-		src={object('src', Config, src)}
-		style={{
-			position: 'absolute',
-			width: ri.scale(select('orientation', prop.orientation, Config) === 'vertical' ? 768 : 1020),
-			height: ri.scale(select('orientation', prop.orientation, Config) === 'vertical' ? 588 : 240)
-		}}
-	>
-		{text('children', Config, 'ImageItem Caption')}
-	</ImageItem>
+	<div style={{width: ri.scaleToRem(800), height: ri.scaleToRem(600)}}>
```
Comment (MikyungKim) [inline samples/sampler/stories/default/ImageItem.js:26]: Let's not add this div.

File: samples/sampler/stories/default/ImageItem.js [hunk 2]
```diff
@@ -23,22 +23,19 @@ export default {
...
-		centered={boolean('centered', Config)}
-		disabled={boolean('disabled', Config)}
-		label={text('label', Config, 'ImageItem label')}
-		orientation={select('orientation', prop.orientation, Config)}
-		selected={boolean('selected', Config)}
-		showSelection={boolean('showSelection', Config)}
-		src={object('src', Config, src)}
-		style={{
-			position: 'absolute',
-			width: ri.scale(select('orientation', prop.orientation, Config) === 'vertical' ? 768 : 1020),
```
Comment (MikyungKim) [inline samples/sampler/stories/default/ImageItem.js:36]: Please just change the `ri.scale` to `ri.scaleToRem` in this sampler.

**Review by MikyungKim — CHANGES_REQUESTED** [review]
Please merge develop and put the change log at the right place :)

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**